### ISSUE TITLE
Crash on scan 

### ISSUE THREAD

**Issue by benwaffle** [csicar/Ning#90]
App version: 2.0.0

```
FATAL EXCEPTION: main
Process: de.csicar.ning, PID: 31167
java.lang.IllegalStateException: Room cannot verify the data integrity. Looks like you've changed schema but forgot to update the version number. You can simply fix this by increasing the version number.
	at androidx.room.RoomOpenHelper.checkIdentity(RoomOpenHelper.java:154)
	at androidx.room.RoomOpenHelper.onOpen(RoomOpenHelper.java:135)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.onOpen(FrameworkSQLiteOpenHelper.java:142)
	at android.database.sqlite.SQLiteOpenHelper.getDatabaseLocked(SQLiteOpenHelper.java:428)
	at android.database.sqlite.SQLiteOpenHelper.getWritableDatabase(SQLiteOpenHelper.java:317)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper$OpenHelper.getWritableSupportDatabase(FrameworkSQLiteOpenHelper.java:92)
	at androidx.sqlite.db.framework.FrameworkSQLiteOpenHelper.getWritableDatabase(FrameworkSQLiteOpenHelper.java:53)
	at androidx.room.SQLiteCopyOpenHelper.getWritableDatabase(SQLiteCopyOpenHelper.java:90)
	at androidx.room.RoomDatabase.beginTransaction(RoomDatabase.java:352)
	at de.csicar.ning.ScanDao_Impl$2.call(ScanDao_Impl.java:50)
	at de.csicar.ning.ScanDao_Impl$2.call(ScanDao_Impl.java:47)
	at androidx.room.CoroutinesRoom$Companion$execute$2.invokeSuspend(CoroutinesRoom.kt:54)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at androidx.room.TransactionExecutor$1.run(TransactionExecutor.java:45)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:919)
```

**Comment by csicar** [comment]
Thank you for the bug report. Should be fixed in version 2.0.1

As a workaround, you should be able to clear the app data in the android settings.

**Comment by csicar** [comment]
Closing this, as version 2.0.1 should fix it
